### PR TITLE
remove protractor test infrastructure

### PR DESCRIPTION
**Pull request by megahirt**
## Description

This removes Docker and npm infrastructure for protractor tests.  We will wait to delete the `/test/app` directory for a later PR.

### Type of Change

- chore

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have reviewed the title & description of this PR which I will use as the squashed PR commit message
- [x] I have enabled auto-merge (optional)

### CODE REVIEW

**Review by myieye — DISMISSED** [review]

**Comment by myieye** [comment]
This is a beautiful PR 😁

File: Makefile [hunk 1]
```diff
@@ -9,14 +9,14 @@ start: build
 dev: start
 	docker compose up -d ui-builder
 
-.PHONY: playwright-tests-ci
```
[the remark this answers is not in the document]
Comment (megahirt) [reply Makefile:12]: good call!

[the remark this answers is not in the document]
Comment (longrunningprocess) [reply Makefile:12]: looks like this was done to me, I'm going to resolve this so it can go in.

**Review by longrunningprocess — APPROVED** [review]

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (megahirt) [reply Makefile:12]: done in latest commit

**Pull request merged**